src/components/va-alert/va-alert.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'va-alert',
+  styleUrl: 'va-alert.css',
+  shadow: true,
+})
+export class VaAlert {
+  /* Determines the icon and border/background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
+
+  /**
+   * If true, renders the alert with only a background color corresponding
+   * to the status - no icon or left border.
+   */
+  @Prop() backgroundOnly: boolean = false;
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  @Prop() visible: boolean = true;
+
+  /**
+   * Headline for the alert. Won't show if `backgroundOnly` is true
+   */
+  @Prop() headline: string;
+
+  /**
+   * Determines the level of the heading in the alert
+   */
+  @Prop() level: number = 3;
```

Comment:
I think the better description is nice, but if we did this we would also have to change it in `<va-accordion>`. Additionally, camelCased props have to be converted to kebab case even in JSX, so we would have to make sure all developers understand that.

![image](https://user-images.githubusercontent.com/2008881/115630216-770db000-a2b8-11eb-806f-5398e6d7bf9e.png)

Reply:
I'm not concerned about the kebab casing. We can't stick to only single-word prop names, so that's fine. I can make a ticket for updating the accordion component prop to match—`headingLevel`. That is, unless we wanna go the named slots route, which I think I like. Gets rid of two props (`headline` and `level`).

(I also find it mildly ironic that `onClose` is camel case in the screenshot there. :joy: )